src/map/clif.c

```diff
@@ -5488,6 +5489,7 @@ void clif_skill_memomessage(struct map_session_data* sd, int type)
 ///     0 = "Unable to Teleport in this area" in color 0xFFFF00 (cyan)
 ///     1 = "Saved point cannot be memorized." in color 0x0000FF (red)
 ///     2 = "This skill cannot be used within this area." in color 0xFFFF00 (cyan)
+///     3 = "This item cannot be used within this area." in color 0xFFFF00 (cyan)
```

Comment:
is old clients support this flag too?

Reply:
Fixed by adding the client provided by you, also I added a message in message.conf